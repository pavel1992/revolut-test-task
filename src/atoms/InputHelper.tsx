import styled from 'styled-components';

import { colors } from '../styleConstants/colors';
import { TABLET_BREAKPOINT } from '../styleConstants/mediaConstants';

export const InputHelper = styled.span`
  font-size: 72px;
  line-height: 98px;
  color: ${colors.black};
  @media(max-width: ${TABLET_BREAKPOINT}) {
    font-size: 64px;
  }
`;
