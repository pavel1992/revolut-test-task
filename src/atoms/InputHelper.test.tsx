import 'jest-styled-components';
import * as React from 'react';
import renderer from 'react-test-renderer';

import { InputHelper } from './InputHelper';


test('renders InputHelper correctly', () => {
  const render = renderer
    .create(<InputHelper>111</InputHelper>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
