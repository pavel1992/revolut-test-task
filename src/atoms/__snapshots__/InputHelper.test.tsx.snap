// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders InputHelper correctly 1`] = `
<span
  className="sc-AxjAm iiqNAN"
>
  111
</span>
`;
